client/src/modules/CreateForm/CreateForm.tsx

```diff
@@ -1,28 +1,35 @@
 import { ChangeEvent } from 'react';
+import { useAppSelector } from '../../hooks/redux-hooks';
 import withFormSubmit from '../../hocs/withFormSubmit';
 import { withFormFullPropTypes } from '../../types';
 import { formatFormDate } from '../../services/formatDate';
 import styles from './CreateForm.module.scss';
 
 const CreateForm = (props: withFormFullPropTypes) => {
+  const { container, form, input, button } = styles;
   const { text, setText, created, setCreated, expire, setExpire, onSubmit } = props;
 
-  const minDate = formatFormDate();
+  const currentItem = useAppSelector(state => state.todoListReducer.activeItem);
+
+  const minDate = currentItem ? formatFormDate(currentItem.created) : formatFormDate();
```

Comment:
Try formatFormDate(currentItem?.created), and check in function what you get

Reply:
Ok, thanks.